packages/radix-ui-themes/src/styles/tokens/color.css

```diff
@@ -1218,7 +1288,12 @@
   --accent-a12: var(--gray-a12);
 }
 
-/* gray color overrides */
+/* * * * * * * * * * * * * * * * * * * */
+/*                                     */
+/*           Map gray color            */
+/*                                     */
+/* * * * * * * * * * * * * * * * * * * */
+
 .radix-themes {
   &:where([data-gray-color='mauve']) {
```

Comment:
Unrelated to the PR, but why do we need `.radix-themes:where([data-gray-color='mauve'])` for grays, but it's just `[data-accent-color='tomato']` for accents? Can we do `[data-gray-color='mauve']` directly?